src/test/java/net/openhft/chronicle/core/JvmTest.java

```diff
@@ -58,7 +58,7 @@ public void checkThreadDump() {
 
     @Test
     public void addToClassPath() {
-        Jvm.addToClassPath(JvmTest.class);
+        assertDoesNotThrow(() -> Jvm.addToClassPath(JvmTest.class));
```

Comment:
why add assertDoesNotThrow? I can't see the point. If that code throws then the test will fail anyway

Reply:
Each test should have an explicit assert. This explicitly shows the purpose of the test. 

Otherwise, it is hard to distinguish implicit assertions from omitted assertions. That is the main argument.